src/components/Form/Form.tsx

```diff
@@ -0,0 +1,126 @@
+import { useState } from 'react';
+import { User } from '../../types/User';
+import { TodoExtended } from '../../types/TodoExtended';
+import { findUserById } from '../../utils/findUserById';
+import './Form.scss';
+
+type Props = {
+  user: User[];
```

Comment:
Why is it a user if it's actually an array?

Reply:
> Why is it a user if it's actually an array?

Because of building test cases. I also don't like it